src/hooks/useAuth.tsx

```diff
@@ -4,7 +4,7 @@ import { setSignOut, setUser } from "store/slices/userSlice";
 import { RootState, userPersistor } from "store/store";
 
 const useAuth = () => {
-  const { authToken, email, role } = useSelector(
+  const { authToken, email, role, id } = useSelector(
```

Comment:
Since we can't store the id in localStorage we have to also remove it from the useAuth() hook